metadata-ingestion/src/datahub/sql_parsing/schema_resolver.py

```diff
@@ -73,6 +74,17 @@ def __init__(
             extra_columns={"is_missing": lambda v: v is None},
         )
 
+    def __enter__(self) -> "SchemaResolver":
+        return self
+
+    def __exit__(
+        self,
+        exc_type: Optional[Type[BaseException]],
+        exc_value: Optional[BaseException],
+        traceback: Optional[TracebackType],
+    ) -> None:
+        self.close()
```

Comment:
it inherits from Closeable, so this is not required

Reply:
done